static/src/javascripts/projects/common/modules/audio/utils.js

```diff
@@ -22,4 +24,28 @@ const range = (min: number, max: number) => {
     return ret;
 };
 
-export { format, formatTime, range };
+const sendToOphan = (id: string, eventName: string) => {
+    ophan.record({
+        audio: {
+            id,
+            eventType: `audio:content:${eventName}`,
+        },
+    });
+};
+
+const checkForTimeEvents = (id: string, percent: number) => {
+    if (percent === 25) {
+        sendToOphan(id, '25');
+    }
```

Comment:
it's a small thing but you can set each subsequent `if` as the alternative clause to the previous one with `else`

Reply:
or a switch?

Reply:
yes good point - I'll go for `else`